### PR TITLE
Update and cleanup Readme.md

### PR DESCRIPTION
**Pull request by rschuppan**
Improved usability, readability, and cleanup of older points.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,21 +47,13 @@
 
 ![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
 
-## About The Project
-
-[![Glimpse][product-screenshot]](https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor)
```
Comment (YuvrajRaghuvanshiS) [inline README.md:52]: Placeholder for demo video.

Comment (YuvrajRaghuvanshiS) [reply README.md:52]: Needed.

File: README.md [hunk 2]
```diff
@@ -47,21 +47,13 @@
...
 
 The idea is to install "Legacy Version" of WhatsApp on you device via adb and use "adb backup"  to fetch files from "/data/data/com.whatsapp" folder which includes both the 'key' and 'msgstore.db' (non encrypted) file and after that restore current WhatsApp.
 
-![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
-
-### Built With
-* [Python](https://www.python.org/)
-* [Bash](https://www.gnu.org/software/bash/) (for Linux and OS X)
-#### Depends upon    
-* [Java](https://www.java.com/) (To extract backup)
```
Comment (YuvrajRaghuvanshiS) [inline README.md:64]: Reduce visibility.

File: README.md [hunk 3]
```diff
@@ -75,28 +67,29 @@ After [intallation](#installation) follow on screen instructions.
...
 
-* O/S: Any windows/Mac/Linux. Do not have access to any of these? Try [Termux Edition.](https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor/tree/termux) 
+* O/S: Compatible and tested are Windows & Linux. Mac should be possible as well. Do not have access to any of these? Try [Termux Edition.](https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor/tree/termux) 
 * [Python 3.x](https://www.python.org/downloads/)
 * [Java](https://www.java.com/en/download/)
 * [ADB Drivers](https://developer.android.com/studio/releases/platform-tools) 
-* USB Debugging must be enabled on the target device. Settings -> Developer Options -> (Debugging) USB debugging  
-     If you cannot find Developer Options then please go to: Settings -> About phone/device and tap the Build number multiple times until you're finally declared a developer.  
+* USB Debugging must be enabled on the target Android device. Settings -> Developer Options -> (Debugging) USB debugging  
+     If you cannot find Developer Options then please go to: Settings -> About phone/device and tap the Build number 10 times until you're declared a developer.
```
Comment (YuvrajRaghuvanshiS) [inline README.md:75]: 7 times, really no need to change.

File: README.md [hunk 4]
```diff
@@ -75,28 +67,29 @@ After [intallation](#installation) follow on screen instructions.
...
 * Android device with Android 4.0 or higher. i.e. Ice Cream Sandwich, Jelly Bean, KitKat, Lollipop, Marshmallow, Nougat, Oreo, Pie, Q.  
 
 
 ![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
 
 ### Installation
+Enter the follwing code into the terminal of your choice, on Windows for example cmd.exe. Remember to got into the folder of the code first (via cd). 
 
-1. Clone the repo
+1. Clone / download the repo
```
Comment (YuvrajRaghuvanshiS) [inline README.md:84]: This is how clone is done. Download option is up-top on repo.

File: README.md [hunk 5]
```diff
@@ -105,6 +98,8 @@ sudo ./bin/linuxdependencies.sh
 ```python
 python3 wa_kdbe.py
 ```
+4.1 "Choose device from 'List of devices attached'": Here you have to copy-paste the name of your device from the devices list above.
```
Comment (YuvrajRaghuvanshiS) [inline README.md:101]: That's good.

File: README.md [hunk 6]
```diff
@@ -116,7 +111,6 @@ List of devices attached
 0123a4b5678	device
 emulator-5554 unauthorized
 ```
-* Choose device from "List of devices attached" : 0123a4b5678
```
Comment (YuvrajRaghuvanshiS) [inline README.md:119]: That was needed.

File: README.md [hunk 7]
```diff
@@ -125,12 +119,9 @@ emulator-5554 unauthorized
 
 ![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
 
-## Roadmap
+## Roadmap and ToDo
 
 See the [open issues](https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor/issues) for a list of proposed features (and known issues).
-#### ToDo
-* ![Status](https://img.shields.io/badge/Status-Done-success) Zip extracted folder with password.
-* ![Status](https://img.shields.io/badge/Status-Removed-red) Command line arguments
```
Comment (YuvrajRaghuvanshiS) [inline README.md:133]: Why'd you do that.

File: README.md [hunk 8]
```diff
@@ -47,21 +47,13 @@
...
 The idea is to install "Legacy Version" of WhatsApp on you device via adb and use "adb backup"  to fetch files from "/data/data/com.whatsapp" folder which includes both the 'key' and 'msgstore.db' (non encrypted) file and after that restore current WhatsApp.
 
-![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
-
-### Built With
-* [Python](https://www.python.org/)
-* [Bash](https://www.gnu.org/software/bash/) (for Linux and OS X)
-#### Depends upon    
-* [Java](https://www.java.com/) (To extract backup)
+It is build with [Python](https://www.python.org/) and [Bash](https://www.gnu.org/software/bash/) (for Linux and OS X) and depends on [Java](https://www.java.com/) (to extract backup).
```
Comment (YuvrajRaghuvanshiS) [inline README.md:56]: Reduces visibility.

File: README.md [hunk 9]
```diff
@@ -75,28 +67,29 @@ After [intallation](#installation) follow on screen instructions.
...
 
 ![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
 
 ### Installation
+Enter the follwing code into the terminal of your choice, on Windows for example cmd.exe. Remember to got into the folder of the code first (via cd). 
 
-1. Clone the repo
+1. Clone / download the repo
 ```bash
-git clone https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor.git && cd WhatsApp-Key-Database-Extractor
```
Comment (YuvrajRaghuvanshiS) [inline README.md:93]: you can run further command with "cd"-ing into directory first.

**Comment by YuvrajRaghuvanshiS** [comment]
Sorry could not merge. I only found one changes that was needed to be changed. I commited that in  6665ce082bbd164cfd8fefec618aa1952f526b55  and mentioned this PR and you. I'm closing this PR, hope you don't mind.

File: README.md (hunk 1 above)
Comment (rschuppan) [reply README.md:52]: ah ok

File: README.md (hunk 3 above)
Comment (rschuppan) [reply README.md:75]: ok

File: README.md (hunk 4 above)
Comment (rschuppan) [reply README.md:84]: Yes I know that NOW. 
But as someone who never works with Github, this improves understanding / usability a GREAT deal, for everyone who isn't a regular Github user.

File: README.md (hunk 7 above)
Comment (rschuppan) [reply README.md:133]: They're not really todo als they're both done. And I thought closed ToDos are more or less under issues anyway.

File: README.md (hunk 4 above)
Comment (YuvrajRaghuvanshiS) [reply README.md:84]: That is very basic and I believe adding extra to that confuses user. This is first step for every newbie.

File: README.md (hunk 7 above)
Comment (YuvrajRaghuvanshiS) [reply README.md:133]: Yes, here they are also serving purpose of history and roadmap of project. One does not need to search through issue for them.

**Pull request closed without merging**